compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/backhandler/DesktopBackGestureDispatcher.kt

```diff
@@ -26,18 +26,18 @@ import androidx.compose.ui.input.key.type
 @OptIn(ExperimentalComposeUiApi::class)
 internal class DesktopBackGestureDispatcher: BackGestureDispatcher() {
     fun onKeyEvent(event: KeyEvent): Boolean {
-        if (event.type == KeyEventType.KeyUp && event.key == Key.Escape) {
-            onBack()
-            return true
+        if (event.type == KeyEventType.KeyDown && event.key == Key.Escape) {
+            return onBack()
         } else {
             return false
```

Comment:
IDEA suggests to lift the `return` outside the if.

Reply:
I believe it is easier to read when you see the explicit return near with the function call